calcom/cal.com
# fix: issues with verified emails in workflow

## What does this PR do?

`getVerifiedEmails` returned all team members emails as verified, but `verifyEmailSender` didn't which caused issues. 

Now both look for: the user's email, team members emails, verified emails

## How should this be tested?

- Create team workflow 
- Choose send email to specific email 
- Type in the logged in user's email, this should be automatically verified 
- Click save, workflow should be saved with the verified user's email
- A new entry for verifiedEmail should be created, so email will show as verified also when other team members view the workflow
